skrl/agents/torch/ppo/ppo.py

```diff
@@ -219,8 +227,9 @@ def act(self, states: torch.Tensor, timestep: int, timesteps: int) -> torch.Tens
             return self.policy.random_act({"states": self._state_preprocessor(states)}, role="policy")
 
         # sample stochastic actions
-        actions, log_prob, outputs = self.policy.act({"states": self._state_preprocessor(states)}, role="policy")
-        self._current_log_prob = log_prob
+        with torch.autocast(device_type=self._device_type, enabled=(self._mixed_precision)):
```

Comment:
Why `self._mixed_precision` in all `enabled=(self._mixed_precision)` statements is between `()`?

Reply:
it's unnecessery (legacy from testing). removed.